app/javascript/src/Stats/buyer/stats_application.jsx

```diff
@@ -16,12 +23,24 @@ export class StatsApplicationSourceCollector extends StatsSourceCollector {
   getSources (options) {
     let id = options.selectedApplicationId
     let selectedMetricName = options.selectedMetricName
```

Comment:
We could take the chance to renew or code:
`const { id, selectedMetricName } = options`